libs/storage/include/storage/key.hpp

```diff
@@ -70,9 +70,7 @@ struct Key
 
   Key() = default;
 
-  // TODO(private issue 957): There are a number of implicit conversions for this key, in many
```

Comment:
Let’s make sure this issue is marked as resolved

Reply:
Done